File: java/messages/src/main/java/com/nedap/retail/messages/article/Price.java
package com.nedap.retail.messages.article;

import java.io.Serializable;

public class Price implements Serializable {

    private static final long serialVersionUID = 7065318648199132728L;

    public String currency;
    public String region;
    public Double amount;

    public Price() {
    }

    public Price(final String currency, final String region, final Double amount) {
        this.currency = currency;
        this.region = region;
        this.amount = amount;
    }

    @Override
    public String toString() {
        return "[" + region + "|" + currency + "|" + amount + "]";
    }
}

File: csharp/NedapRetailApiClient/Nedap.Retail.Api/Article.V2/Price.cs
using Newtonsoft.Json;

namespace Nedap.Retail.Api.Article.V2
{
    /// <summary>
    /// Describes a price for a region. These prices are numeric with typically double precision.
    /// </summary>
    public class Price : JsonPrintableObject
    {
        /// <summary>
        /// The Currency for the specified region.
        /// Currency noted according to the ISO 4217:2008 standard.
        /// Example XEU for Euro.
        /// </summary>
        [JsonProperty("currency")]
        public string Currency { get; set; }

        /// <summary>
        /// The Country defining the region where this size description is used.
        /// Country noted according to the ISO 3166:2013 standard.
        /// Example NL for Netherlands.
        /// </summary>
        [JsonProperty("region")]
        public string Region { get; set; }

        /// <summary>
        /// Amount of monetary value.
        /// </summary>
        [JsonProperty("amount")]
        public double Amount { get; set; }
    }
}